src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int DEFAULT_CAPACITY = 12;
+    private static final int EMPTY_VALUE = -1;
+    private static final int FULL_VALUE = DEFAULT_CAPACITY - 1;
+
+    private final int[] numbers = new int[DEFAULT_CAPACITY];
+    private int total = EMPTY_VALUE;
+
+    public int getTotal() {
+		return total;
+	}
+
+	public void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    public boolean callCheck() {
+        return total == EMPTY_VALUE;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.